packages/itwinui-css/backstop/scenarios/index.js

```diff
@@ -15,10 +15,28 @@ fs.readdirSync('backstop/scenarios/').forEach((file) => {
   scenarios = scenarios.concat(newScenarios);
 });
 
+const lightScenarios = scenarios.map((s) => {
+  const lightScenario = Object.assign({}, s);
```

Comment:
Just curious, the spread operator does not work here to create a clone?

Reply:
I'm not sure. I copied this from darkScenarios, which was already here.